proto/ibc/applications/fee/v1/query.proto

```diff
@@ -0,0 +1,123 @@
+syntax = "proto3";
+
+package ibc.applications.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "ibc/applications/fee/v1/fee.proto";
+import "google/api/annotations.proto";
+import "ibc/core/channel/v1/channel.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// Query provides defines the gRPC querier service.
+service Query {
+  // Gets the fee expected for submitting ReceivePacket msg for the given packet
+  rpc ReceiveFee(QueryReceiveFeeRequest) returns (QueryReceiveFeeResponse) {
+    option (google.api.http).get =
+        "/ibc/apps/fee/v1/receive_fee/port/{packet_id.port_id}/channel/{packet_id.channel_id}/sequence/"
+        "{packet_id.sequence}/relayer/{relayer_address}/height/{query_height}";
```

Comment:
I think we should leave off the relayer address and query height so that they are [query parameters](https://github.com/aspnet/AspLabs/blob/c1e59cacf7b9606650d6ec38e54fa3a82377f360/src/GrpcHttpApi/sample/Proto/google/api/http.proto#L90)

I think query height is pretty naturally a query parameter and I think it is useful to have the relayer address as a query parameter. This can allow default behaviour to query incentivized packets which are unassigned (can be relayed by any relayer). I imagine for legal reasons, some relayers won't want to query for packets which have assigned relayers even if they are an assigned relayer